Resources/Prototypes/Catalog/Fills/Boxes/security.yml

```diff
@@ -54,3 +54,22 @@
   - type: Item
   - type: Storage
     capacity: 20
+
+- type: entity
+  name: ziptie box
+  parent: BoxBase
+  id: BoxZiptie
+  description: A box full of zipties.
+  components:
+  - type: StorageFill
+    contents:
+      - id: Zipties
+        amount: 10
```

Comment:
This is a lot of entities for something that should probably be a stack?

Reply:
this is consistent with handcuff boxes iirc

Reply:
Can probably just make an issue to have deferred storagefill or something.

Reply:
Yeah I copied handcuffs 1:1, does it need changing?

Reply:
> does it need changing?

Well its not exclusive to this box so however its solved it will probably be more general & apply to other boxes/storage so I don't think there's anything you need to do here.